resources/js/components/Listing/Filters/CustomSelected.vue

```diff
@@ -0,0 +1,59 @@
+<script>
+import { swatches } from '../../../stores/useSwatches'
+
+export default {
+    props: {
+        selectedValues: {
+            type: Object,
+            default: () => {},
+        },
+    },
+
+    render() {
+        return this.$scopedSlots.default(this)
+    },
+
+    computed: {
+        activeFilters() {
+            return Object.keys(this.selectedValues)
+                .filter((filterKey) => {
+                    return (
+                        this.selectedValues[filterKey].showFilter &&
+                        (Array.isArray(this.selectedValues[filterKey].value)
+                            ? this.selectedValues[filterKey].value.length
+                            : !!this.selectedValues[filterKey].value)
+                    )
+                })
+                .flatMap((filterKey) => {
+                    let result
+
+                    if (
+                        Array.isArray(this.selectedValues[filterKey].value) &&
+                        !['category', 'price', ...Object.keys(swatches.value)].includes(filterKey)
```

Comment:
Can't we do something with early returns or a switch case so we don't have to specify this list here?